composer.json

```diff
@@ -40,7 +40,7 @@
     },
     "require-dev": {
         "doctrine/coding-standard": "^11.0",
-        "doctrine/data-fixtures": "^1.7",
+        "doctrine/data-fixtures": "^1.8 || ^2.0",
```

Comment:
The support for LoggerInterface was added in [1.8.0](https://github.com/doctrine/data-fixtures/releases/tag/1.8.0) by https://github.com/doctrine/data-fixtures/pull/462

Please update the conflict rule too.

Reply:
Yup, already changed that. Shouldn't `doctrine/data-fixtures` be a regular dependency instead of a dev dependency? What's the reasoning behind making it a dev dependency?

Reply:
Not all projects use fixtures. This bundle configure them only if the `doctrine/data-fixtures` package is required.

Reply:
Right, thank you for the explanation, makes perfect sense!